app/src/features/authentication/public/utils/apolloRefresh.ts

```diff
@@ -1,31 +1,34 @@
-import { createRefresh } from 'react-auth-kit';
+import Cookies from 'js-cookie';
+import createRefresh from 'react-auth-kit/createRefresh';
 
 import { client } from '~/core/stores/apollo';
-import { Mutation, MutationRefreshTokenArgs } from '~/core/types/generated/graphql';
+import { RefreshTokenMutation, RefreshTokenMutationVariables } from '~/core/types/generated/graphql';
 import { REFRESH_TOKEN } from '~/features/authentication/graphql/authentication.gql';
 
-// TODO: :not-sure-if: Not sure if this is working :hmmm:
+export const refreshApiCallback = async () => {
```

Comment:
I'm pretty sure...this is obsolete. Let me check

Reply:
(As in like. It wasn't being used before this and was a remnant that's never called)